cmd/update-notifier.go

```diff
@@ -60,7 +74,7 @@ func colorizeUpdateMessage(updateString string, newerThan time.Duration) string
 
 	// Box cannot be printed if terminal width is small than maxContentWidth
 	if maxContentWidth > termWidth {
-		return "\n" + line1InColor + "\n" + line2InColor + "\n" + "\n"
+		return fmt.Sprint("\n", line1InColor, "\n", line2InColor, "\n\n")
```

Comment:
1. Is it good idea to make
```
>
>  {line1InColor}
>  {line2InColor}
>
```
2. As all values are strings, I am not sure why `fmt.Sprint()` is used over `+`
3. You could use string array and join for better readability.